boards/arm/mimxrt1015_evk/doc/index.rst

```diff
@@ -0,0 +1,170 @@
+
+  - 128 Mbit QSPI Flash
+
+- Connectivity
+
+  - Micro USB host and OTG connectors
+  - Arduino interface
+
+- Audio
+
+  - Audio Codec
+  - 4-pole audio headphone jack
+  - External speaker connection
+  - Microphone
+
+- Debug
+
+  - JTAG 10-pin connector
+  - OpenSDA with DAPLink
+
+For more information about the MIMXRT1015 SoC and MIMXRT1015-EVK board, see
+these references:
+
+- `i.MX RT1015 Website`_
+- `i.MX RT1015 Datasheet`_
+- `i.MX RT1015 Reference Manual`_
+- `MIMXRT1015-EVK Website`_
+- `MIMXRT1015-EVK Quick Reference Guide`_
+- `MIMXRT1015-EVK Design Files`_
+
+Supported Features
+==================
+
+The mimxrt1015_evk board configuration supports the following hardware
+features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+``boards/arm/mimxrt1015_evk/mimxrt1015_evk_defconfig``
+
+Other hardware features are not currently supported by the port.
+
+Connections and I/Os
+====================
+
+The MIMXRT1015 SoC has five pairs of pinmux/gpio controllers.
+
++---------------+-----------------+---------------------------+
+| Name          | Function        | Usage                     |
++===============+=================+===========================+
+| GPIO_AD_B0_05 | GPIO            | LED                       |
++---------------+-----------------+---------------------------+
+| GPIO_EMC_09   | GPIO            | SW4                       |
++---------------+-----------------+---------------------------+
+| GPIO_AD_B0_06 | LPUART1_TX      | UART Console              |
++---------------+-----------------+---------------------------+
+| GPIO_AD_B0_07 | LPUART1_RX      | UART Console              |
++---------------+-----------------+---------------------------+
+
+System Clock
+============
+
+The MIMXRT1015 SoC is configured to use the 24 MHz external oscillator on the
+board with the on-chip PLL to generate a 500 MHz core clock.
+
+Serial Port
+===========
+
+The MIMXRT1015 SoC has eight UARTs. ``LPUART1`` is configured for the console,
+and the remaining are not used.
+
+Programming and Debugging
```

Comment:
Please see #14000 for an update to this section.

Reply:
Okay, I'll update it according to that. What shall I put for the opensda firmware? It's not available yet.

Reply:
tested with the commit of 641a245.